cunumeric/linalg/cholesky.py

```diff
@@ -202,12 +202,37 @@ def tril(context: Context, p_output: StorePartition, n: int) -> None:
     task.execute()
 
 
+def batched_cholesky(output: DeferredArray, input: DeferredArray) -> None:
+    # the only feasible implementation for right now is that
+    # each cholesky submatrix fits on a single proc. We will have
+    # wildly varying memory available depending on the system.
+    # Just use a fixed cutoff to provide some sensible warning.
+    # TODO: find a better way to inform the user dims are too big
+    size = input.base.shape[-1]
+    if size > 32768:
```

Comment:
how this number is chosen? I think it might depend on architecture and type of the data, right?

Reply:
Also, I think this should be a warning instead

Reply:
Converted to a warning and  added clarifying comments